crates/proto/src/rr/domain/name.rs

```diff
@@ -198,7 +198,11 @@ impl Name {
         Self::from_str(name).or_else(|_| Self::from_ascii(name))
     }
 
-    fn from_encoded_str<E: LabelEnc>(local: &str, origin: Option<&Self>) -> ProtoResult<Self> {
+    fn from_encoded_str<E: LabelEnc>(
+        local: &str,
+        origin: Option<&Self>,
+        //allow_underscore: bool,
```

Comment:
Leftover commented code?

Reply:
It's actually scaffolding for my next commit, where I was going to introduce `from_host_utf8()` and `from_host_ascii()` (naming to be bikeshedded) that retain the stricter semantics.